data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1020 @@
+			`	When they return, they are holding in one of their arms a small spherical device which they hand to the alien who seems to be in charge, who then in turn offers it to you and burbles an unintelligible phrase with what you might have characterized as an encouraging tone. Several individuals in the crowd by now appear to be taking pictures of the two of you, and you can see more of them in the distance gliding and swimming towards your landing site.`
+			choice
+				`	(Take the device.)`
+				`	(Refuse.)`
+					goto refused
+			`	You take the device from the alien. It liquifies the moment it touches your skin before freezing in mid-splash to project a small but detailed holographic star map of the surrounding systems. A star system a few jumps east of here is highlighted amidst a sea of alien lettering. The individual standing opposite you grunts energetically as the trio of interceptors begins to hover in preparation for takeoff: it would appear that these people, whoever they are, want you to travel there.`
+				accept
+			label refused
+			`	Thinking the device might be a weapon of some sort, you back away from the alien offering it to you and into the safety of the <ship>'s hatch. Though none of the aliens make any move to harm you, you notice from inside that after a couple of minutes, water slowly begins to flow onto the landing pad. It might be wise to take off sooner rather than later, so you do so sooner.`
+				flee
+
+	npc accompany save
+		government "House Sioeora"
+		personality timid launching escort
+		ship "Stolsaqra (Mission Escort)" "Kijet-i-khisa"
+		ship "Stolsaqra (Mission Escort)" "Iyiq-sora"
+		ship "Stolsaqra (Mission Escort)" "Seqra-saqra"
+
+	on fail
+		dialog `At least one of your escorts have been destroyed. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+
+	on visit
+		conversation
+			`You have reached <planet>, but without your escorts, you are refused landing permission at the spaceport and must return to orbit until they arrive.`
+				launch
+
+	on complete
+		event "label successor space"
+		event "successors: first contact wait" 5
+		log "Factions" "Successors" `The Successors are a society of aliens occupying a territory on the southeastern outer rim of the galaxy. They are the heirs of a large interstellar empire which collapsed at some point around fifteen hundred years ago, after exploring much of the galaxy, even as far north as Sol.`
+		log "People" "Sieasej" `Sieasej is a Successor scholar of historical linguistics and member of the noble House Sioeora. Because of their area of study, they are one of the only Successors who can speak to and understand you without the benefit of a translation device.`
```

Comment:
The _intentional_ lore is that the Predecessors (Successors' ancestors) explored as far north as Sol in the ~1500s, did some spying (or "cultural observation") on humans then, but retreated back to their core territories and evacuated soon after as the space around their core worlds started to fall apart. Since then, they haven't gone any farther than one can go by hyperdrive from their space.
Sieasej is the equivalent of a medieval history scholar that learned how to speak 5th century Frankish. I like the idea that there are modern Successors who retain an interest in humans for the same reason that there are modern humans who retain an interest in dragons: they're both these weird charismatic megafauna that people talked about a long time ago, but **no**, there's no Successors spying on the Republic or whatnot---they don't know it exists unless FL tells them.

Reply:
Sorry, I thought I had deleted this comment after reading further...

Yes, your explanation makes perfect sense.